db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180406080400) do
+ActiveRecord::Schema.define(version: 20180508070232) do
```

Comment:
Sanity check: we need to rebase https://github.com/kubic-project/velum/blob/master/packaging/suse/patches/0_set_default_salt_events_alter_time_column_value.rpm.patch when the schema changes.